### PR TITLE
Add function to count and find problems with missing tests

### PR DESCRIPTION
**Pull request by raiman264**
## What was added
An output to the console when you run all the tests so you can see how many problems are there and if there is any missing Test

Thanks to this I figured out that 2 of the test files had wrong naming

example output:
Screenshot 2024-10-11 at 9 06 21 p m

## Checklist before requesting a review
- [x] I have checked the [guide](https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript#contributions) for contributions.
- [ ] I have added the description at the top of the solution file.
- [ ] I have added a test file for the problem.
- [ ] I have added an entry to the [README](https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript/blob/master/README.md) file with a link to the new file.
- [x] I have run linter (`npx eslint LeetcodeProblems LeetcodeProblemsTests`) and unit tests (`node Test.js`) and they pass. [More info](https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript#run-tests).

Check the [guide](https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript#contributions) and [this PR](https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript/pull/39) example.

### CODE REVIEW

File: Test.js [hunk 1]
```diff
@@ -2,34 +2,45 @@
...
+  let problems = [];
+  for(const i in paths) {
+    const folder = paths[i];
+    const new_problems = await loadProblemsFiles(folder); // await
+    problems = problems.concat(new_problems);
+  }
+  return problems;
+};
+
+const test_all = async function () {
```
Comment (ignacio-chiazzo) [inline Test.js:29]: Could you rename the following functions?

- `test_all` -> `runAllTests`
- `solve` -> `solveProblem`

Also, let's use `camelCase` everywhere. I noticed we have a mix. A linter rule could solve this too (not need to do it here).

File: Test.js [hunk 2]
```diff
@@ -65,10 +76,40 @@ var loadProblemsFiles = (folder) => {
...
+    console.table(missingTests);
+  }
+};
+
 if (process.argv.length > 2) {
   const path = process.argv.pop();
   solve(path);
 } else {
   test_all();
+  get_missing_tests();
```
Comment (ignacio-chiazzo) [inline Test.js:113]: Both call `get_all_tests`, we could load them outside the class and pass them as arguments:

```js
  const tests = await get_all_tests(TESTS_FOLDERS);
  const problems = await get_all_tests(PROBLEMS_FOLDERS);
  
  run_all_tests(tests)
  get_missing_tests(tests, problems)
```

**Review by ignacio-chiazzo — COMMENTED** [review]
Love it! Thanks!

File: Test.js (hunk 1 above)
Comment (raiman264) [reply Test.js]: updated all variables and functions in this file to be `camelCase`
I also added the eslint rule to make it visible and noticed there are a lot of issues

I can create another PR to solve all of those linter issues

File: Test.js (hunk 2 above)
Comment (raiman264) [reply Test.js]: good catch, updated ;)

**Review by ignacio-chiazzo — APPROVED** [review]
Nice! thank you. 

Screenshot 2024-10-15 at 7 34 46 PM

File: Test.js (hunk 1 above)
Comment (ignacio-chiazzo) [reply Test.js]: Love it! that'd be amazing.

**Pull request merged**